flink-connector/flink/src/main/java/org/apache/gravitino/flink/connector/store/GravitinoCatalogStore.java

```diff
@@ -87,8 +94,10 @@ public Optional<CatalogDescriptor> getCatalog(String catalogName) throws Catalog
           CatalogDescriptor.of(catalogName, Configuration.fromMap(flinkCatalogProperties));
       return Optional.of(descriptor);
     } catch (Exception e) {
-      LOG.warn("Failed to get the catalog:{}", catalogName, e);
-      return Optional.empty();
+      throw new CatalogException(
```

Comment:
Modify it like this?

`throw new CatalogException(String.format("Failed to get the catalog: %s ", catalogName), e);`